content/webapp/views/components/WorksSearchResults/WorksSearchResults.Result.tsx

```diff
@@ -25,13 +24,6 @@ type Props = {
   resultPosition: number;
 };
 
-// TODO: remove, hack to handle the fact that we are pulling through PDF thumbnails.
-// These will be removed from the API at some stage.
-function isPdfThumbnail(thumbnail: DigitalLocation): boolean {
-  // e.g. https://dlcs.io/iiif-img/wellcome/5/b28820769_WG_2006_PAAG-implementing-persistent-identifiers_EN.pdf/full/!200,200/0/default.jpg
-  return Boolean(thumbnail.url.match('.pdf/full'));
-}
-
```

Comment:
Unrelated to this PR (sort of, I _was_ going to use it and moved it around) but we decided to just remove this https://wellcome.slack.com/archives/C3TQSF63C/p1764001849619849

I can split it out if we prefer, sorry!